### PR TITLE
Tu 36 UI display menu

### PR DESCRIPTION
**Pull request by KatePolyakov**

### CODE REVIEW

File: src/api/ApiLinks.jsx [hunk 1]
```diff
@@ -1 +1,4 @@
-export const CuisinesURL = `${process.env.REACT_APP_BACKEND_URL}/api/cuisines/getAllCuisines`;
\ No newline at end of file
+export const CuisinesURL = `${process.env.REACT_APP_BACKEND_URL}/api/cuisines/getAllCuisines`;
```
Comment (anithaamarnath) [inline src/api/ApiLinks.jsx:1]: @KatePolyakov can u merger main and change the folder accordingly 
please thank

File: src/components/Menu/Menu.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+//Icons
+import Star from '../../icons/star-svgrepo-com.svg';
+import Info from '../../icons/info-circle-svgrepo-com.svg';
+
+import classes from './menu.module.scss';
+
+const Menu = () => {
+  const [menuList, setMenuList] = useState([]);
+
+  useEffect(() => {
```
Comment (anithaamarnath) [inline src/components/Menu/Menu.jsx:17]: this should move to services folder

**Review by anithaamarnath — CHANGES_REQUESTED** [review]

File: src/api/ApiLinks.jsx (hunk 1 above)
Comment (anithaamarnath) [reply src/api/ApiLinks.jsx:1]: also Delete this ApiLinks file

File: src/components/Menu/Menu.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    axios.get(MenuURL).then((response) => setMenuList(response.data));
+  }, []);
+
+  return (
+    <div>
+      {menuList.map((restaurants) => (
+        <div key={restaurants._id}>
+          {restaurants._id === '651e32d67a5bb9de20aaf7ec' ? (
```
Comment (anithaamarnath) [inline src/components/Menu/Menu.jsx:25]: No harded  code the ID  and follow the main branch folder structure eg: Login Componement

**Review by anithaamarnath — CHANGES_REQUESTED** [review]

File: src/components/Menu/Menu.jsx (hunk 2 above)
Comment (KatePolyakov) [reply src/components/Menu/Menu.jsx:17]: did it

**Comment by KatePolyakov** [comment]
TU-36-UI-On click restuarant-display menu

**Review by anithaamarnath — APPROVED** [review]

**Review by anithaamarnath — APPROVED** [review]

**Review by TeamUp3Batch — APPROVED** [review]

**Review by anithaamarnath — APPROVED** [review]

**Pull request merged**